src/omv/common/sensor_utils.c

```diff
@@ -1124,7 +1124,7 @@ __weak int sensor_auto_crop_framebuffer()
         return 0;
     }
 
-    if (sensor.pixformat == PIXFORMAT_RGB565) {
+    if ((sensor.pixformat == PIXFORMAT_RGB565) || (sensor.pixformat == PIXFORMAT_YUV422)) {
```

Comment:
```suggestion
    if (sensor.pixformat.bpp == 2) {
```
Same as above, will cover future cases.

Reply:
True

Reply:
Not possible to fix as PIXFORMAT is not a struct. You implemented it as an ENUM so you can't get the bpp without shifting and masking. If I do this it makes the code brittle.

Reply:
If you'd like to update the pixformat enum to be a struct so this can be done then this can be fixed. The check should be for is_color and bpp == 2.